VideoLocker/res/layout/fragment_discussion_comments.xml

```diff
@@ -10,9 +10,7 @@
         android:id="@+id/discussion_comments_listview"
         android:layout_width="match_parent"
         android:layout_height="0dp"
-        android:layout_weight="1"
-        android:paddingLeft="@dimen/discussion_responses_standard_margin"
-        android:paddingRight="@dimen/discussion_responses_standard_margin" />
+        android:layout_weight="1" />
```

Comment:
Again, note that this layout seems to be functionally identical to `fragment_discussion_responses`, and we could probably just use _one_ of these instead.